### DISCUSSION TITLE
subst alias not mounted

### DISCUSSION THREAD

**Discussion by gcattan** [containers/podman#23797] [Q&A] [answered]
Hi, 
I am using `subst` to associate a path (e.g. C:/test) with a drive letter (let's call it J:)
When I am starting my podman container, this letter is mounted under `/mnt/j`, but there is nothing under j.
(C:/test is correctly mounted under /mnt/c/test)

Is there a way to automatically mount everything under the alias J:/ to /mnt/j?

**Comment by Luap99** [comment, marked as answer]
We do not mount anything explicitly for WSL this is all handled by windows not us so I recommend you ask on some windows/WSL forum.
